notebooks_tsqr/NightLog.ipynb

```diff
@@ -19,7 +19,8 @@
     "# Times Square replaces this cell with the user's parameters.\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    from lsst.summit.utils import ConsDbClient
</pre><p>ConsDB specialized code should be refactored into source_adapters.py</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
This is an external-to-our-repository package: https://github.com/lsst-sitcom/summit_utils/blob/main/python/lsst/summit/utils/consdbClient.py